### PR TITLE
[gardening] Daily cleanup: typos, header formatting, pep8 fixes.

### PR DESCRIPTION
**Pull request by practicalswift**
#### What's in this pull request?

Daily cleanup: typos, header formatting, pep8 fixes.
#### Resolved bug number: –

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

**Comment by gottesmm** [comment]
@swift-ci Please smoke test

**Comment by gribozavr** [comment]
@swift-ci Please smoke test

**Comment by gribozavr** [comment]
@practicalswift 

```
Traceback (most recent call last):
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/test/Prototypes/../../utils/gyb", line 3, in <module>
    gyb.main()
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/utils/gyb.py", line 1125, in main
    args.target.write(execute_template(ast, args.line_directive, **bindings))
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/utils/gyb.py", line 1015, in execute_template
    ast.execute(execution_context)
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/utils/gyb.py", line 619, in execute
    x.execute(context)
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/utils/gyb.py", line 702, in execute
    result = eval(self.code, context.local_bindings)
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/test/Prototypes/Integers.swift.gyb", line 42, in <module>
    class struct(object):
  File "/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/swift/test/Prototypes/Integers.swift.gyb", line 49, in struct
    struct(operator='+', name='adding',      mutatingName='add',           firstArg='',           llvmName='add', kind='+'),
NameError: name 'struct' is not defined
```

**Comment by practicalswift** [comment]
@gribozavr I'm afraid I cannot reproduce this issue. `utils/build-script -cRT` runs fine on my machine.

**Comment by practicalswift** [comment]
FWIW, the change made to `test/Prototypes/Integers.swift.gyb` is indentation only :-)

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@practicalswift Sorry, it seems to reproduce consistently in CI.  To make progress, would you mind reverting changes to that file?

File: test/Prototypes/Integers.swift.gyb [hunk 1]
```diff
@@ -45,21 +45,20 @@ class struct(object):
   def __repr__(self):
     return 'struct(%r)' % self.__dict__
 
-binaryArithmetic = [
-       struct(operator='+', name='adding',      mutatingName='add',           firstArg='',           llvmName='add', kind='+'),
-       struct(operator='-', name='subtracting', mutatingName='subtract',      firstArg='',           llvmName='sub', kind='-'),
-       struct(operator='*', name='multiplied',  mutatingName='multiply',      firstArg='by',         llvmName='mul', kind='*'),
-       struct(operator='/', name='divided',     mutatingName='divide',        firstArg='by',         llvmName='div', kind='/'),
-       struct(operator='%', name='remainder',   mutatingName='formRemainder', firstArg='dividingBy', llvmName='rem', kind='/')
+  binaryArithmetic = [
```
Comment (rintaro) [inline test/Prototypes/Integers.swift.gyb]: @practicalswift  I don't know why your local test passes.
This change modifies the scope of `binaryArithmetic`.

```python
class Foo(object):
    def __init__(self):
        pass

    foo = Foo()
```
```
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    class Foo(object):
  File "test.py", line 6, in Foo
    foo = Foo()
NameError: name 'Foo' is not defined
```

**Comment by practicalswift** [comment]
@gribozavr, I've now reverted the `.gyb` file change. Sorry, I accidentally introduced a scoping error. Thanks for @rintaro to noticing!

Turns out this test is not run under Linux (due to `// REQUIRES: OS=macosx`) so that is why it wasn't triggered by my own testing. Sorry :-)

**Pull request merged**